api/src/opentrons/config/feature_flags.py

```diff
@@ -84,3 +84,11 @@ def flex_stacker_tof_sensors_disabled() -> bool:
     return advs.get_setting_with_env_overload(
         "disableFlexStackerLabwareDetection", RobotTypeEnum.FLEX
     )
+
+
+def camera_enabled() -> bool:
+    return advs.get_setting_with_env_overload("enableCamera", RobotTypeEnum.FLEX)
```

Comment:
```suggestion
def camera_enabled(robot_type: RobotTypeEnum) -> bool:
    return advs.get_setting_with_env_overload("enableCamera", robot_type)
```
Since this impacts both the OT-2 and Flex.